apps/frontend/src/components/global/RegistrationModal.vue

```diff
@@ -216,6 +216,7 @@ export default class RegistrationModal extends Vue {
             SnackbarModule.notify(
               'You have successfully registered a new user'
             );
+            ServerModule.FetchAllUsers();
```

Comment:
fetch all users returns a promise, but it's not properly chained nor written in async/await notation.